mne/preprocessing/tests/test_ica.py

```diff
@@ -608,4 +608,27 @@ def test_fit_params():
     ICA(fit_params=fit_params)  # test no side effects
     assert_equal(fit_params, {})
 
+
+@requires_sklearn
+def test_bad_channels():
```

Comment:
I added a new test method that simply calls `ica.fit` when an EOG channel is present. The method feels a bit off compared to the other test ones that are implemented, because it does not call an `assert` anywhere. The reason for that is that I don't know what I would compare the resulting value with since ICA is not deterministic right?